daemon/compress.c

```diff
@@ -0,0 +1,75 @@
+
+#include <str.h>
+#include <zlib.h>
+#include <string.h>
+#include <errno.h>
+
+#include "log.h"
+#include "udp_listener.h"
+#include "str.h"
+#include "compress.h"
+
+#define windowBits 15
```

Comment:
While this is defined after the zlib.h inclusion, it matches the name of the function argument. I'd use something more descriptive and less generic, perhaps `#define RTPENGINE_ZLIB_WINDOWBITS 15`?